docs/participate/the-community/social-media.md

```diff
@@ -1,34 +1,70 @@
 ---
 title: Social Media
-description: IOTA has a threiving and active community on all popular social media applications. Find the IOTA community on your prefered medium.
+description: IOTA has a threiving and active community on all popular social media applications. Find the IOTA Foundation and the community here.
 ---
 
 
-# Heading 1 (Page Title)
+# IOTA Social Media
 
-content - introduction to the topic
+The IOTA project has established a social media presence on all popular social media channels. These are the presences handled by the IOTA Foundation:
 
+## Discord
 
-## Heading 2 (Sub topic 1)
+This is without doubt the most active community and with nearly 50.000 members a great place for everyone that wants to learn more about IOTA and get to know the people involved. A lot of IOTA Foundation Team members and also the Founders and members of the Board are active on Discord and take part in the community discussions and projects there. Every aspect of the project is covered here in separate channels and you can find the whole range of this amazing community from serious mathematics and cryptographic topic up to memes and Fun channels. 
+Please join us there and get to know the amazing community behind the project by following this invite Link:
+[IOTA Discord](https://discord.iota.org)
+ 
 
-content 
+## Twitter
 
-## Heading 2 (Sub topic 2)
+The official IOTA Foundation Twitter Account is this one here, which will provide the newest developments, announcements, and information regarding the IOTA Project:
 
-content
+[IOTA Foundation Twitter](https://twitter.com/iota)
 
+The engineering Team of the IOTA Foundation also runs its own Twitter account for Technologie updates:
```

Comment:
Mabe it's just me, but could you rephrase this sentence? Technologie updates doesn't sound right to me 🤔

Reply:
Yeah good point - its "denglishing" too much - ill rewrite that